library/build.gradle

```diff
@@ -13,7 +13,7 @@ ext {
     junit_jupiter_version = '5.9.1'
     junit_platform_launcher_version = '1.9.1'
 
-    mockito_version = '4.11.0'
+    mockito_version = '4.10.0'
```

Comment:
why this downgrade? was this on purpose?

Reply:
Thank you for pointing it out, uniforming to the latest version through the entire project